bundles/org.eclipse.swt/Readme.Linux.md

```diff
@@ -32,12 +20,20 @@ You need to install the following on your system:
 
 Import and build the SWT project `bundles/org.eclipse.swt` and `binaries/org.eclipse.swt.gtk.linux.${arch}` in Eclipse.
 
-To build SWT natives run the `Build-SWT-native-binaries-for-running-platform` Maven launch configuration.
-Alternatively run the `build.sh` script in `binaries/org.eclipse.swt.gtk.linux.${arch}/bin/library` directory. To
-use the locally build natives run the build script with the 'install' argument
-(`./build.sh install`) or --help to see more options.
+See **Building and Testing locally** in [Readme.md](Readme.md)
+
+Alternatively run the `build.sh` script under `org.eclipse.swt\Eclipse SWT PI\gtk\library\` directory. 
+To use the locally build natives run the script with the 'install' and `clean` arguments (`./build.sh clean install`)
+or --help to see more options.
+
+The JDK against which the SWT natives are built must be specified in the `SWT_JAVA_HOME` environment variable:
+  ```
+  SWT_JAVA_HOME=<YOUR_JDK_PATH>
+  export SWT_JAVA_HOME
+  ```
```

Comment:
I'm not so much a Linux export, but from the existing build scripts I think that's the right thing to do.
Nevertheless it would be great if someone with more knowledge could confirm this.

Reply:
Yes, it is correct to set `SWT_JAVA_HOME` (this way).

Still, the instruction do not work as the whole setup relies on being executed in a properly configured binary fragment project and not within the general `org.eclipse.swt` project:
* The default `OUTPUT_DIR` specification in `build.sh` assumes that the `build.sh` is _not_ executed from within the `org.eclipse.swt` project but from without the symlink created for the binaries fragment project, see #1127.
* There are further files required (like `make_common.mak`), which will only be available when properly linked together in a OS-/WS-/Arch-specific fragment project.

So the instruction have to be rewritten to be executed from within a binaries fragment, but then you need to describe as well how to set up that fragment (create the symlink etc, which is automatically done by Eclipse or the Maven build, respectively). Since it is very easy to create the binaries via Maven, I am not sure whether it is worth to describe all that manual setup (and keep it up to date), as that is already done by Maven (and thus documented in the Maven configuration). Whoever does not want to use Maven can just replicate by hand what Maven does (as that's what has to be described in these instructions otherwise).

Reply:
I removed all the manual steps in d556545966b1beefd598027b20675dfa34cde7e6.

@HannesWell we can add a permanent link or other further improvements after this PR ;-)